source/features/better-profile-readme-text.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import cache from 'webext-storage-cache';
+import * as pageDetect from 'github-url-detection';
+
+import * as api from '../github-helpers/api';
+
+import features from '.';
+
+const getReadmePath = cache.function(
+	async (owner: string, repository: string): Promise<string> => {
+		const {name: readme} = await api.v3(
+			`repos/${owner}/${repository}/readme`
+		);
+		return readme;
+	}
+);
+
+async function init(): Promise<void> {
+	const title = select(
+		'#js-pjax-container > div.container-xl.px-3.px-md-4.px-lg-5 > div > div.flex-shrink-0.col-12.col-md-9.mb-4.mb-md-0 > div:nth-child(2) > div > div.Box.mt-4 > div > div > div'
+	);
```

Comment:
Don't use this kind of selector, it means nothing and it's really fickle. Prefer
```suggestion
	const link = select('.user-profile-nav .octicon-smiley + a');
```